sycl/test/basic_tests/kernel_bundle/kernel_bundle_api.cpp

```diff
@@ -0,0 +1,96 @@
+// RUN: %clangxx -fsycl -fsycl-device-code-split=per_kernel -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %t.out
```

Comment:
According to post-commit testing this test doesn't work on the host device, so we either should move it to `on-device` directory or fix execution on the host device.

Reply:
The intention of the spec, of course, is that these APIs would work on all devices.